src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -311,15 +406,12 @@ public DescriptorImpl getDescriptor() {
 
     /**
      * Descriptor for {@link DockerBuilder}. Used as a singleton.
-     * The class is marked as public so that it can be accessed from views.
-     *
-     * <p>
-     * See <tt>src/main/resources/hudson/plugins/hello_world/DockerBuilder/global.jelly</tt>
-     * for the actual HTML fragment for the plugin global config screen.
      */
     @Extension
     public static final class DescriptorImpl extends BuildStepDescriptor<Builder> {
 
+        // Using docker-commons now, methods left for backwards compatibility
```

Comment:
Seems like a false compatibility. Rather than getting `NoSuchMethodError`, hypothetical callers from another plugin (are there any?) would get a `null` return value (after the second restart after the upgrade), so they would still not really work.